evtx/Maps/Application_System Restore_8194.map

```diff
@@ -0,0 +1,41 @@
+Author: Hyun Yi @hyuunnn
+Description: restore point created successfully
```

Comment:
Restore, not restore. Can you capitalize that first word?

Reply:
@rathbuna Fixed :)